### ISSUE TITLE
Perl-5.34.1-GCCcore-11.3.0.eb fails to download Variable-Magic-0.62.tar.gz

### ISSUE THREAD

**Issue by gunnarwustl** [easybuilders/easybuild-easyconfigs#17199]
Attempting to install Perl-5.34.1-GCCcore-11.3.0.eb, explicitly or as a dependency, fails to complete due to the Variable-Magic-0.62.tar.gz file no longer existing at the source_url provided.

This appears to be due to the upstream maintainer updating the package to version 0.63 while no longer maintaining, at least at the same location, a 0.62 tar.gz any longer.

The results of running `eb -r Perl-5.34.1-GCCcore-11.3.0.eb:
```bash
== Temporary log file in case of crash /tmp/723159.tmpdir/eb-5zhe4loo/easybuild-_zpovq3s.log
== found valid index for /home/easybuild/.local/easybuild/software/EasyBuild/4.6.1/easybuild/easyconfigs, so using it...
== found valid index for /home/easybuild/.local/easybuild/software/EasyBuild/4.6.1/easybuild/easyconfigs, so using it...
== resolving dependencies ...
== processing EasyBuild easyconfig /home/easybuild/.local/easybuild/software/EasyBuild/4.6.1/easybuild/easyconfigs/p/Perl/Perl-5.34.1-GCCcore-11.3.0.eb
== building and installing Perl/5.34.1-GCCcore-11.3.0...
== fetching files...
== ... (took 1 secs)
== FAILED: Installation ended unsuccessfully (build directory: /tmp/easybuild/build/Perl/5.34.1/GCCcore-11.3.0): build failed (first 300 chars): Couldn't find file Variable-Magic-0.62.tar.gz anywhere, and downloading it didn't work either... Paths attempted (in order): /home/easybuild/.local/easybuild/software/EasyBuild/4.6.1/easybuild/easyconfigs/p/Perl/p/Perl/extensions/Variable-Magic-0.62.tar.gz, /home/easybuild/.local/easybuild/software/ (took 2 secs)
== Results of the build can be found in the log file(s) /tmp/723159.tmpdir/eb-5zhe4loo/easybuild-Perl-5.34.1-20230124.155413.zqxYS.log
ERROR: Build of /home/easybuild/.local/easybuild/software/EasyBuild/4.6.1/easybuild/easyconfigs/p/Perl/Perl-5.34.1-GCCcore-11.3.0.eb failed (err: "build failed (first 300 chars): Couldn't find file Variable-Magic-0.62.tar.gz anywhere, and downloading it didn't work either... Paths attempted (in order): /home/easybuild/.local/easybuild/software/EasyBuild/4.6.1/easybuild/easyconfigs/p/Perl/p/Perl/extensions/Variable-Magic-0.62.tar.gz, /home/easybuild/.local/easybuild/software/")
```

A screenshot of the page at the following url: https://www.cpan.org/authors/id/V/VP/VPIT/
![image](https://user-images.githubusercontent.com/114174587/214343678-a6534176-aada-4f79-908e-3f756bba69e7.png)

**Comment by jfgrimm** [comment]
@gunnarwustl this was fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/16611

**Comment by boegel** [comment]
Indeed, please update to EasyBuild v4.7.0, or work around the problem using `eb --from-pr 16611 ...`

**Comment by gunnarwustl** [comment]
@boegel I have installed EasyBuild, per the documentation, via EasyBuild. Currently there is no EasyBuild-4.7.0.eb available in the 4.7.x repos. This results in the effective version available for use being v4.6.2, as this is the latest version available in the format of an easyconfig.

Are there plans to add the 4.7.0 easyconfig such that it can be installed via EasyBuild? Or would you simply recommend not installing EasyBuild in this fashion anymore? Rather that users should simply install EasyBuild via pip or from source and use it as normal.

**Comment by jfgrimm** [comment]
@gunnarwustl the recommended way to install the latest easybuild release is to use `eb --install-latest-eb-release`
